database/project.js

```diff
@@ -0,0 +1,22 @@
+var Sequelize = require('sequelize');
+
+module.exports = function(db) {
+  var Project = db.define('Project', {
+    id: {
+      type: Sequelize.UUID,
+      allowNull: false,
+      primaryKey: true
+    },
+    name: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      unique: true
+    },
+    metadata: {
+      type: Sequelize.JSONB,
+      allowNull: false
```

Comment:
Is it better to not allow nulls, or just default to an empty object?

Reply:
@batpad that's a great point. I'll swap `defaultValue` in for `allowNull` for all optional properties.